Rawl_B502/bunad_laigen.ttl

```diff
@@ -11,142 +11,183 @@
-    rel:ancestorOf <#Eterscéla>.
+    irishRel:genName "Lugdach";
+    rel:ancestorOf <#NuaduNecht>;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#LugaidLuath>
 
 <#FergusFairrce>
     a foaf:Person ;
-    irishRel:nomName "Fergus Fairrce" .
+    irishRel:nomName "Fergus Fairrce" ;
+    rel:childOf <#NuaduNecht-9d624370>;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#FergusFairrcge>.
+
+<#NuaduNecht-9d624370>
+    a foaf:Person;
+    irishRel:nomName "Nuadu Necht";
+    owl:sameAs <#NuaduNecht>.
 
 <#Airt>
     a foaf:Person ;
     irishRel:genName "Airt" ;
     irishRel:nomName "Art" ;
-    rel:ancestorOf [
-        a foaf:Person ;        
-    ] .
-        
+    rel:ancestorOf <#NuaduNecht>.
+
 <#SétnaiSithbaicc>
     a foaf:Person ;
     irishRel:genName "Sétnai Sithbaicc" ;
-    irishRel:nomName "Sétna Sithbacc" .
+    irishRel:nomName "Sétna Sithbacc" ;
+    rel:ancestorOf <#NuaduNecht>;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#Sétna>.
 
-<#Bricc>
-    a foaf:Person ;
-    irishRel:genName "Bricc" ;
-    irishRel:nomName "Brecc" ;
-    rel:ancestorOf [
-        a foaf:Person
-    ] .
-        
-<#Bressail>
+<#BressailBricc>
     a foaf:Person ;
-    irishRel:genName "Bressail" ;
-    irishRel:nomName "Bressal" .
+    irishRel:genName "Bressail Bricc" ;
+    irishRel:nomName "Bressal Brecc" ;
+    owl:sameAs <#Bressail>;
+    rel:ancestorOf <#NuaduNecht>;
+    rel:childOf <http://example.com/Rawl_B502/fursunnud_naile.ttl#BresalBrecc>.
+
+<#Lugaid-db5d2810>
+    a foaf:Person;
+    irishRel:nomName "Lugaid";
+    rel:ancestorOf <#NuaduNecht>.
 
 <#Sedrach>
     a foaf:Person ;
-    irishRel:accName "Sedrach" .
+    irishRel:accName "Sedrach" ;
+    rel:ancestorOf <#NuaduNecht>.
+
+# Could <#Lugaid-db5d2810> and <#Sedrach> be the same person
+# (= <http://example.com/Rawl_B502/fursunnud_naile.ttl#LugaidLuath>)? EPT
+
+<#Bressal>
+    a foaf:Person;
+    irishRel:nomName "Bressal";
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#BresalBrecc>;
+    owl:sameAs <#BressailBricc>;
+    rel:ancestorOf <#NuaduNecht>.
 
 <#Fiachrai>
     a foaf:Person ;
     irishRel:genName "Fiachrai" ;
-    irishRel:nomName "Fiachra" .
+    irishRel:nomName "Fiachra" ;
+    rel:ancestorOf <#NuaduNecht>;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#FiachuFobrecc>.
 
 <#Ailill>
     a foaf:Person ;
-    irishRel:nomName "Ailill" .
+    irishRel:nomName "Ailill" ;
+    rel:ancestorOf <#NuaduNecht>;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#AilillGlass>.
 
 <#Foglass>
     a foaf:Person ;
-    irishRel:nomName "Foglass" .
+    irishRel:nomName "Foglass" ;
+    rel:ancestorOf <#NuaduNecht>;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#FeradachFoglas>.
 
-<#Nuadu>
+<#NuaduFuildon>
     a foaf:Person ;
-    irishRel:nomName "Nuadu" ;
-    rel:childOf <#Fuildon>.
+    irishRel:nomName "Nuadu Fuildon" ;
```

Comment:
Technically yes. Good point. A king called Nuadu Fuildon appears elsewhere in this and other manuscripts, sometimes also called Alldóit. The line here is "Nuadu Fuildon forfích fiansa fóensius" ("Nuadu Fuildon who conquers warbands, who overthrows"), which is admittedly ambiguous (the original ms. lacking commas).

But, equally, there is no evidence that Nuadu and Fuildon are father and son, at this point in the manuscript or elsewhere. If we were to apply this principle generally, we should be splitting all the names which include sobriquets up into two people. Nuadu Necht is Nuadu son of Necht; Cairbre Lifechair is Cairbre son of Lifechair; Cú Chulainn is Cú son of Chulainn etc. This might seem absurd, but, if we can't make some reference to other sources, who can argue against it?  

Can we agree that we can make reference to other testimony (giving priority to the ms. at hand) in cases of ambiguous readings, but not to "correct" something stated unambiguously, however incongruous? So, if the text had actually said "Nuadu mac Fuidon" at this point, I would not be allowed to bring in appearances of "Nuadu Fuildon" in Rawl.B.502 or beyond to amend this.